website/docs/core-concepts/building-ui/dynamic-ui/README.md

```diff
@@ -46,3 +46,9 @@ Get Category Dropdown (Default Option property)
 Here Table1 is the name of the widget
 
 ![Click to expand](</img/form_-_table.gif>)
+
+## [Height](./../../../reference/widgets/README.md#height)
```

Comment:
The heading height doesn't go well with the rest of the doc here. I think `Updating widget height`.